### PR TITLE
Stackdriver in Cromwell Perf [BA-5729]

### PR DESCRIPTION
**Pull request by salonishah11**
This PR enables sending metrics to Stackdriver in Cromwell Perf (and removes StatsD configuration).

### CODE REVIEW

File: services/src/main/scala/cromwell/services/instrumentation/impl/stackdriver/StackdriverInstrumentationServiceActor.scala [hunk 1]
```diff
@@ -165,7 +165,8 @@ object StackdriverInstrumentationServiceActor {
     /**
       * Transforms a CromwellBucket to a Stackdriver path
       */
-    def toStackdriverString = (CromwellMetricPrefix ++ cromwellBucket.prefix ++ cromwellBucket.path.toList).mkString("/").replace(" ", "_")
+    def toStackdriverString = (CromwellMetricPrefix ++ cromwellBucket.prefix ++
+        cromwellBucket.path.toList).mkString("/").replaceAll(" ", "_").replaceAll("\\[|\\]", "")
```
Comment (aednichols) [inline services/src/main/scala/cromwell/services/instrumentation/impl/stackdriver/StackdriverInstrumentationServiceActor.scala]: I believe you can do `raw"\[|\]"` to potentially increase clarity

File: scripts/perf/vm_scripts/cromwell/cromwell.conf [hunk 2]
```diff
@@ -70,13 +70,14 @@ backend {
...
     config {
-      hostname = statsd-proxy
-      port = 9125
-      prefix = ${?CROMWELL_STATSD_PREFIX} # can be used to prefix all metrics with an api key for example
-      flush-rate = 5 seconds # rate at which aggregated metrics will be sent to statsd
+      auth = "application-default"
+      google-project = ${?CROMWELL_PROJECT}
+      # rate at which aggregated metrics will be sent to Stackdriver API, must be 1 minute or more.
+      flush-rate = 1 minute
+      cromwell-perf-test-case = ${?TEST_CASE_DIRECTORY}
```
[the remark this answers is not in the document]
Comment (kshakir) [reply scripts/perf/vm_scripts/cromwell/cromwell.conf]: The syntax (re-)defines the field only if the expression in `${?expression}` is defined.

https://github.com/lightbend/config/blob/master/HOCON.md#substitutions

[the remark this answers is not in the document]
Comment (cjllanwarne) [reply scripts/perf/vm_scripts/cromwell/cromwell.conf]: I guess I was more curious what the `cromwell.conf` is doing with a `cromwell-perf-test-case` value in the first place

**Comment by cjllanwarne** [comment]
Look good in the separated perf tests so 👍 from me. @salonishah11 - I'll merge this for you so that I can get started  with horizontalling the performance tests.

**Pull request merged**